tests/pages/base.py

```diff
@@ -3,11 +3,11 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 from selenium.webdriver.common.by import By
-from selenium.webdriver.support import expected_conditions as expected
 from selenium.webdriver.support.select import Select
 from selenium.webdriver.support.ui import WebDriverWait as Wait
 
 from .page import Page, PageRegion
+from .newsletter_embed import NewsletterEmbedForm, MozillaNewsletterEmbedForm
```

Comment:
In the past we've use a `regions` module for locating page regions. Otherwise these may appear to be full page objects.